src/rpc/methods/eth.rs

```diff
@@ -1840,7 +2015,46 @@ mod test {
             )
             .unwrap(),
         );
-        assert_eq!(expected_hash, eth_tx_args.hash().unwrap());
+        assert_eq!(expected_hash, tx.eth_hash().unwrap());
+    }
+
+    #[test]
+    fn test_rlp_encoding_legacy() {
+        let tx = EthTx {
+            r#type: EIP_LEGACY_TX_TYPE.into(),
+            chain_id: 314159.into(),
```

Comment:
Where is this test case coming from? Can we add a link to the filscan/filfox transaction?

Reply:
Fixed.